Dockerfile

```diff
@@ -130,9 +130,12 @@ RUN . ../ENV/bin/activate \
     && pip install -e . --no-binary :all:
 
 # Install Yarn components
-COPY ./package.json ${ARCHES_ROOT}/package.json
```

Comment:
This copy command was initially used to run yarn install before copying the complete code base into the container (COPY . ${ARCHES_ROOT}). The advantage of that is you don't have to run yarn install every time you make little changes to the code base (unless package.json changes).

Reply:
There has been some changes to where and how this is done in master. https://github.com/archesproject/arches/pull/4878 which broke the automated setup in docker and the entrypoint. The line is not necessary, as package.json is now in the arches/install directory and copied over in a previous command, along with the yarnrc, etc all within the install dir again.

Reply:
What I meant to say was that in the current order of commands it seems that every time a developer makes a change to any file in the `${ARCHES_ROOT}` folder, the Docker build will have to run the `yarn install` (and also `pip install`) command. This should not be necessary of you run those before doing `COPY . ${ARCHES_ROOT}`. Those steps will then be cached and reused the next time you build the image. 

(And for that to work you need to copy `package.json` separately from `COPY . ${ARCHES_ROOT}` ).

Reply:
So add a copy of package.json, yarn.lock, and .yarnrc to the /arches/install dir and run `yarn install` before the `COPY . ${ARCHES_ROOT}` step?

Reply:
Exactly!

Reply:
I've just pushed the change to move the yarn install higher, to help it be cached as discussed.